### ISSUE TITLE
Reicast crashes when it is in a path with accents

### ISSUE THREAD

**Issue by alaquepasa** [libretro/RetroArch#6934]
Hi, I tested now Reicast (normal, the core that say DreamCast) core and it crashes when the content to load contains accents. I tested this 4 situations, I will write example paths:

- RetroArch + ISO in path without accents (C:/test/RetroArch and C:/test/game): All works fine
- RetroArch + ISO in path with accents (C:/Emulación/RetroArch and C:/Emulación/game): Crash
- RetroArch with (C:/Emulación/RetroArch) and game without (C:/test/game): Crash
- RetroArch without (C:/test/RetroArch) and game with (C:/Emulación/game): It loads only the BIOS

So it seems that this core can't load paths with accents. If he can't load BIOS core crash, and if he can't load the game, it goes to the BIOS.

I'm using RetroARch, dated Jun 27 2018, git 04311695aa on Windows 10 64 bits

**Comment by bparker06** [comment]
Indeed reicast does not use our filestream interface but just generic `fopen()` calls, so non-English paths on Windows will not work properly. Can you create an issue in the reicast repo for this?

https://github.com/libretro/reicast-emulator/issues

Thanks.

**Comment by RobLoach** [comment]
https://github.com/libretro/reicast-emulator/issues/108

**Comment by alaquepasa** [comment]
Is a shame this. With RetroArch 1.6 (previous to VFS) I had no problems, but when VFS was added cores that doesn't implement it give this problems. And yes, I know that previous versions worked because previous versions can load (they can find the BIOS path) and can save in game and save states.

Now I have some cores that can't work on Windows, when previously they did. I did a "workarround" overriding configurations, creating a special "system" "save" path to cores that have this problem, but is curious that this happens since VFS was introduced when old versions worked (and, of course, the "real" emulators works too).
